includes/Settings.php

```diff
@@ -34,99 +34,71 @@
 class Settings extends Service_Base {
 	/**
 	 * Settings group.
-	 *
-	 * @var string
```

Comment:
Do we want these removed?

Reply:
Yes. It's redundant as the type can be inferred from the value below.